extensions/stackdriver/stackdriver.cc

```diff
@@ -352,7 +396,8 @@ void StackdriverRootContext::onTick() {
     }
   }
 
-  if (enableAccessLog()) {
+  if (enableAccessLog() &&
+      (cur - last_log_report_call_nanos_ > log_report_duration_nanos_)) {
```

Comment:
Where is last_log_report_call_nanos_ updated?

Reply:
Thanks..Fixed